src/helpers/xml-builder.js

```diff
@@ -7,8 +7,8 @@
 import { fragment } from 'xmlbuilder2';
 import isVNode from 'virtual-dom/vnode/is-vnode';
 import isVText from 'virtual-dom/vnode/is-vtext';
-import colorNames from 'color-name';
-import { cloneDeep } from 'lodash';
+import colorNames, { cadetblue } from 'color-name';
```

Comment:
Why do we import cadetBlue?

Reply:
my bad. must've accidently imported those due to auto import.
will remove it in the next iteration.